apps/extension/src/background/approvals/service.ts

```diff
@@ -397,7 +397,6 @@ export class ApprovalsService {
 
     if (allowConnection) {
       try {
-        await this.keyRingService.connect(interfaceTabId);
```

Comment:
I've merged to main more approvals tests so something might fail because of this line removal :)